### PR TITLE
feat: thorchain utxo support

### PR DESCRIPTION
**Pull request by elmutt**
## Description

Support all thorchain utxo coins

To test:

Use thorchain to trade out of non-btc utxo coins (LTC, BCH, DOGE)

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Risk

low risk. adds new support for swapper coins 

## Testing

Trade out of a utxo asset that is not bitcoin (LTC for example)

## Screenshots (if applicable)

Screen Shot 2022-08-05 at 11 02 16 AM

### CODE REVIEW

File: src/components/Trade/hooks/useSwapper/useSwapper.ts [hunk 1]
```diff
@@ -614,14 +621,14 @@ export const useSwapper = () => {
       } as unknown as DisplayFeeData<T>
     }
 
-    switch (sellAsset.chainId) {
-      case KnownChainIds.EthereumMainnet:
-      case KnownChainIds.AvalancheMainnet:
+    const { chainNamespace } = fromAssetId(sellAsset.assetId)
+
+    switch (chainNamespace) {
```
Comment (0xApotheosis) [inline src/components/Trade/hooks/useSwapper/useSwapper.ts:626]: Nice refactor.

**Review by 0xApotheosis — COMMENTED** [review]
Does what it says on the box! 

One non-blocking comment on the error messages.

I'm assuming we don't currently handle rendering of trades for UTXOs, but rather show a send and receive TX of the "account" balance coming back in?

Screen Shot 2022-08-12 at 4 26 02 pm

**Review by 0xApotheosis — APPROVED** [review]

**Comment by elmutt** [comment]
> Does what it says on the box!
> 
> One non-blocking comment on the error messages.
> 
> I'm assuming we don't currently handle rendering of trades for UTXOs, but rather show a send and receive TX of the "account" balance coming back in?
> 
> Screen Shot 2022-08-12 at 4 26 02 pm

Correct. no special trade handling or even parsing of the txs. Its just whatever unchained + parser see as the outgoing + incoming txs.

The concept of lableing both thorchain txs as "trades" also doesn't necessarily fit 1-1 to the idea of a zrx trade where its a single tx.

I think the original plan was to come up with some kind of all-encompassing concept of "blockchain activity" that could reference multiple blockchain txs. A type of blockchain activity could be a trade or other.  But thats a discussion we can have another day

**Comment by 0xApotheosis** [comment]
Got it, cheers for the explanation 👌

**Pull request merged**